### PR TITLE
Add CampusModel. Update parsing swipe data mechanism

### PR DESCRIPTION
**Pull request by connorreinhold**
## Overview

Update wait-times parsing to be consistent with the current state of the backend.
Update class structure so wait times can be expanded for all campus eateries, not just dining halls. 

## Changes Made

Added CampusModel.java (where wait times code is located, as extended by both DiningHallModel and CafeModel)
Moved parseSwipeData function to CampusModel.java, called from both DiningHallModel parseEatery and CafeModel's parseEatery method.

## Test Coverage

Just parsed data and logged whether the expected results were seen

## Next Steps 

@YKo20010 should be able to access wait times with mCafeData.getSwipeData() in CampusMenuActivity and from there should be able to fill in the appropriate swipe data

</details>

### CODE REVIEW

File: app/src/main/java/com/cornellappdev/android/eatery/CampusMenuActivity.java [hunk 1]
```diff
@@ -205,7 +206,7 @@ else if (mCafeData instanceof DiningHallModel) {
                 setupViewPager(mCustomPager);
                 mTabLayout.setupWithViewPager(mCustomPager);
                 mTabLayout.setTabTextColors(
-                        ContextCompat.getColor(getApplicationContext(), R.color.primary),
+                        ContextCompat.getColor(getAp plicationContext(), R.color.primary),
```
Comment (ningning621) [inline app/src/main/java/com/cornellappdev/android/eatery/CampusMenuActivity.java]: lmao don't think this is right

File: app/src/main/java/com/cornellappdev/android/eatery/model/CafeModel.java [hunk 2]
```diff
@@ -158,6 +158,8 @@ public void parseEatery(Context context, boolean hardcoded, AllEateriesQuery.Eat
                 .appendPattern("yyyy-MM-dd")
                 .toFormatter(Locale.US);
 
+        parseSwipeData(cafe.swipeData());
```
Comment (ningning621) [inline app/src/main/java/com/cornellappdev/android/eatery/model/CafeModel.java]: hmm, does this not return anything? or where does the data go?

File: app/src/main/java/com/cornellappdev/android/eatery/model/DiningHallModel.java [hunk 3]
```diff
@@ -157,33 +152,12 @@ public void parseEatery(Context context, boolean hardcoded, AllEateriesQuery.Eat
...
-                    timeFormatter);
-            mSwipeDataList.add(
-                    new Swipe(start, end, swipeDatum.swipeDensity(), swipeDatum.waitTimeLow(),
-                            swipeDatum.waitTimeHigh()));
-
-        }
-        Collections.sort(mSwipeDataList);
-        */
-
+        parseSwipeData(eatery.swipeData());
```
Comment (ningning621) [inline app/src/main/java/com/cornellappdev/android/eatery/model/DiningHallModel.java]: same here, how exactly does this method work?

File: app/src/main/java/com/cornellappdev/android/eatery/model/CafeModel.java (hunk 2 above)
Comment (connorreinhold) [reply app/src/main/java/com/cornellappdev/android/eatery/model/CafeModel.java]: Updated this for better styling. The method parseSwipeData basically updated an instance variable but changed it to return a List of Swipes

**Pull request merged**